lib/matplotlib/offsetbox.py

```diff
@@ -1003,8 +1012,9 @@ def draw(self, renderer):
 
 class AnchoredOffsetbox(OffsetBox):
     """
-    An offset box placed according to the legend location
-    loc. AnchoredOffsetbox has a single child. When multiple children
+    A container of an `.OffsetBox` placing it according to location *loc*.
```

Comment:
, placed

Reply:
I wanted to clarify that this is a class for placing another offset box (=child). The fact that it is an offset box as well is secondary. The design seems a bit strange to me, because I would have expected that the child could be any Artist not just an OffsetBox. As it seems, we have to wrap an Artist in an OffsetBox and then again in an AnchoredOffsetBox?!? But I don't care enough about these classes to argue. Reverted.